### ISSUE TITLE
Uncaught TypeError: Converting circular structure to JSON

### ISSUE THREAD

**Issue by cn2156** [vuejs/vue-devtools#69]
ENV:
Lubuntu 15.10
Google Chrome 47.0.2526.106 (64-bit)
Vue.js devtools 1.0.5
Vue1.0.13 Live Mode

Install with:
```
1. Download zip and unzip
2. `npm install`
3. `npm run build`
4. Open Chrome extension page
5. Check "developer mode"
6. Click "load unpacked extension", and choose `shells/chrome`.
```

(1) When go to subroute, the error occured when click the Root node:
```
Uncaught TypeError: Converting circular structure to JSON     extensions::SafeBuiltins:82
```
Details:
```
Save.stringify                       @extensions::SafeBuiltins:82
PortImpl.postMessage                 @extensions::messaging:53
target.(anonymous function)          @extensions::SafeBuiltins:19
publicClass.(anonymous function)     @extensions::utils:94
sendMessageToDevtools                @proxy.js:73
```

(2) Sometime appear two same vue.js devtools, by reopen Chrome dev tools to solve.

**Comment by tdekoning** [comment]
I'm having the same error. In my case it seems that the vue-devtools is trying to parse a circular reference caused by a dom-node that is part of the object on the scope.

I know vue.js itself had a fix a while ago that made sure this does not cause errors. Perhaps the same fix needs to be applied to vue-devtools.

**Comment by dgspeir** [comment]
Experiencing the same issue. 

In my case it was caused by this: 
`this.placesServuce = new google.maps.places.PlacesService(new google.maps.Map(document.createElement('div')));`

Same error and stack trace as original post.

**Comment by yyx990803** [comment]
fixed via 2a626a3

**Comment by yyx990803** [comment]
Side note: you probably don't want to put google maps objects inside `data` - which will cause Vue to recursively convert all its properties (which is wasteful). Just attach it inside `created` hooks.
